spec/abstractops.html

```diff
@@ -101,22 +101,22 @@ <h1>DurationHandleFractions ( _fHours_, _minutes_, _fMinutes_, _seconds_, _milli
         1. If any of _minutes_, _fMinutes_, _seconds_, _milliseconds_, _microseconds_, _nanoseconds_ is not 0, throw a *RangeError* exception.
         1. Let _mins_ be _fHours_ × 60.
         1. Set _minutes_ to floor(_mins_).
-        1. Set _fMinutes_ to _mins_ modulo 1.
+        1. Set _fMinutes_ to the mathematical value whose sign is the sign of _mins_ and whose magnitude is abs(_mins_) modulo 1.
```

Comment:
maybe instead of this, adding a remainder operation alongside modulo? (since % in JS is “remainder”)